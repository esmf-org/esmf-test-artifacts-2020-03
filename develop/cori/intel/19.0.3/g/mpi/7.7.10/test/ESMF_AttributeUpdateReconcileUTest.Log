build time -- 2022-01-31 03:00:43.403819
20220131 165354.559 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220131 165354.559 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220131 165354.559 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220131 165354.559 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220131 165354.559 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220131 165354.559 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220131 165354.559 INFO             PET0 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_06-10-gce27d44403
20220131 165354.559 INFO             PET0 ESMF library build date/time: "Jan 31 2022" "02:56:55"
20220131 165354.559 INFO             PET0 ESMF library build location : /global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220131 165354.559 INFO             PET0 ESMF_COMM                   : mpi
20220131 165354.560 INFO             PET0 ESMF_MOAB                   : enabled
20220131 165354.560 INFO             PET0 ESMF_LAPACK                 : enabled
20220131 165354.560 INFO             PET0 ESMF_NETCDF                 : enabled
20220131 165354.560 INFO             PET0 ESMF_PNETCDF                : disabled
20220131 165354.560 INFO             PET0 ESMF_PIO                    : enabled
20220131 165354.560 INFO             PET0 ESMF_YAMLCPP                : enabled
20220131 165354.565 INFO             PET0  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
20220131 165354.565 INFO             PET0  NUMBER_OF_PROCESSORS           2
20220131 165354.575 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
20220131 165354.576 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
20220131 165354.576 INFO             PET0  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
20220131 165354.576 INFO             PET0 Finalizing ESMF
20220131 165354.559 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220131 165354.559 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220131 165354.559 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220131 165354.559 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220131 165354.559 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220131 165354.559 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220131 165354.559 INFO             PET1 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_06-10-gce27d44403
20220131 165354.559 INFO             PET1 ESMF library build date/time: "Jan 31 2022" "02:56:55"
20220131 165354.559 INFO             PET1 ESMF library build location : /global/u2/r/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220131 165354.559 INFO             PET1 ESMF_COMM                   : mpi
20220131 165354.561 INFO             PET1 ESMF_MOAB                   : enabled
20220131 165354.561 INFO             PET1 ESMF_LAPACK                 : enabled
20220131 165354.561 INFO             PET1 ESMF_NETCDF                 : enabled
20220131 165354.561 INFO             PET1 ESMF_PNETCDF                : disabled
20220131 165354.561 INFO             PET1 ESMF_PIO                    : enabled
20220131 165354.561 INFO             PET1 ESMF_YAMLCPP                : enabled
20220131 165354.565 INFO             PET1  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
20220131 165354.565 INFO             PET1  NUMBER_OF_PROCESSORS           2
20220131 165354.575 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
20220131 165354.576 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
20220131 165354.576 INFO             PET1  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
20220131 165354.576 INFO             PET1 Finalizing ESMF
