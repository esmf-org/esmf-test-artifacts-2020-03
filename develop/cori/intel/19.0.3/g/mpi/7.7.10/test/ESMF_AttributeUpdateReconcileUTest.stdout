build time -- 2022-02-21 03:20:10.216388
  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
  NUMBER_OF_PROCESSORS           2
  NUMBER_OF_PROCESSORS           2
  PASS  PET            0 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
  PASS  PET            1 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
  PASS  PET            0 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
  PASS  PET            1 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
  PET           0  Test Elapsed Time    7.66000000000000       msec.
  PET           1  Test Elapsed Time    10.2590000000000       msec.
